src/udev/udev-builtin-net_id.c

```diff
@@ -45,6 +45,7 @@
  *                                         — PCI geographical location
  *   [P<domain>]p<bus>s<slot>[f<function>][u<port>][..][c<config>][i<interface>]
  *                                         — USB port number chain
+ *   v<slot>                               - VIO slot number
```

Comment:
can you maybe add a reference to "IBM PowerVM" or so to this comment? I for one wouldn't know what VIO is...